third_party/openocd/BUILD.bazel

```diff
@@ -0,0 +1,18 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+package(default_visibility = ["//visibility:public"])
+
+exports_files(glob(["**"]))
+
+genrule(
+    name = "openocd",
+    outs = ["openocd.sh"],
+    cmd = """
+    echo '#!/bin/bash' > $@
+    echo './$(location @openocd//:openocd_bin) $$@' >> $@
```

Comment:
I don't think we need `./` here.

Also, `$(location ...)` is old and busted.  The hot new thing is `$(execpath ...)`.